app/src/main/kotlin/com/wire/android/ui/home/whatsnew/WhatsNewItem.kt

```diff
@@ -51,6 +52,7 @@ fun WhatsNewItem(
     text: String? = null,
     @DrawableRes trailingIcon: Int? = null,
     onRowPressed: Clickable = Clickable(false),
+    placeholder: Boolean = false,
```

Comment:
placeholder means like you need to pass composable, maybe showPlaceholder?

Reply:
It meant that this composable is a placeholder so it should show shimmer effect instead of real data, but I get your point, I changed that to `isLoading` so that it's more clear